pandasql/memory_utils.py

```diff
@@ -0,0 +1,54 @@
+import os
+import subprocess
+import psutil
+from tempfile import mkstemp
+
+import pandas as pd
+import pandasql as ps
+
+
+SAMPLE_ROWS = 1000
+# What proportion of available memory should we actually consider available
+SAFETY_FACTOR = 0.8
```

Comment:
Might want to discuss what we should set SAFETY_FACTOR to for benchmarks on Monday with @raulcf

Reply:
Yep. I think it'll have to be configured after we look at how erroneous our estimations are (i.e., what proportion of false positives we can tolerate).